TestAppUITests/TestAppUITests.m

```diff
@@ -0,0 +1,68 @@
+//
+//  TestAppUITests.m
+//  TestAppUITests
+//
+//  Created by  Sergey Dolin on 23/07/2019.
+//  Copyright © 2019 Calabash. All rights reserved.
+//
```

Comment:
Remove these comments.

Reply:
done

Reply:
UITest target has some problem not easy to fix and can not be build - some files seem to be messed.
Besides i do not think it would be good to mess Calabash and XCTests in the same target: different dependencies, versions, so on. 

Please confirm:
1. i should move XCTest to UITest target and do not fix the build 
2. i should move XCTest to UITest target and try to fix the build
3. keep XCTest and Calabash tests in the separated targets

Reply:
Honestly, I prefer #2 way.
Please try to pull fresh develop to your branch. Probably, your issues will be fixed

Reply:
#324 fixes the build.